Chap_API_Publish.tex

```diff
@@ -195,25 +227,33 @@ \subsection{Range of Published Data}
 Undefined range.
 %
 \declareconstitemvalue{PMIX_RANGE_RM}{1}
-Data is intended for the host environment, or lookup is restricted to data published by the host environment.
+Published data and generated events are restricted to processes executing under the same instance of the host environment as the publisher or event creator.  Lookup of data is restricted to data published by processes running under the same instance of the host environment as the requester.
```

Comment:
Note from 3Q 2022 (Aug. 11, 2022)
 * Old text was intended. That the published data is intended solely for the host environment not for application/tool processes.
   * Suggest leave the old text
 * `PMIX_RANGE_GLOBAL` : Anyone under that RM instance (basically what the new text is saying)